rustls/Cargo.toml

```diff
@@ -26,6 +26,7 @@ webpki = { version = "0.22.0", features = ["alloc", "std"] }
 default = ["logging", "tls12"]
 logging = ["log"]
 dangerous_configuration = []
+extract_secrets = []
```

Comment:
Naming nit: I think I'd prefer `secret_extraction` because the noun form better fits in with something like `dangerous_configuration` (`read_buf` notwithstanding, given that it mirrors the rustc feature name).